### PR TITLE
slicing bank cv modal

### PR DESCRIPTION
**Pull request by maulanamaleek**
- slicing ui (no functionality)
- use data from `Upload Cv` 

Q:
1. table nya punya pagination button, seperti di gambar. apakah oke?

| Result |
|------|
| image|
|image|

### CODE REVIEW

**Comment by maulanamaleek** [comment]
please help to review @dwicaksono / @kalibani

File: components/job/create-job/upload-cv.tsx [hunk 1]
```diff
@@ -52,7 +61,7 @@ const UploadCv = () => {
             <MonitorUp className="mr-2 h-4 w-4" />
```
Comment (dwicaksono) [inline components/job/create-job/upload-cv.tsx:61]: h-4 w-4 next nya bsa kita coba pake size-4 @maulanamaleek (minor)

File: components/table/input-filter.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    label: string
+}
+
+const InputFilter = ({ label, placeholder }: InputFilterProps) => {
+  const [value, setValue] = useState('');
+
+  return (
+    <div>
+    <span className="text-sm">{label}</span>
+      <div className="border rounded-md px-2 py-2 divide-y bg-white/80 backdrop-blur-md flex gap-2 items-center w-fit">
```
Comment (dwicaksono) [inline components/table/input-filter.tsx:17]: px-2 and py-2 bisa pake p-2 (minor)

**Comment by dwicaksono** [comment]
Looks good to me @maulanamaleek 
mungkin bisa di includenya tiket link nya mas ke PR nya dan di tiket jangan lupa screen shoot nya masukin juga.

Q:

table nya punya pagination button, seperti di gambar. apakah oke?

A:
For now, I think it will be fine., wdyt @kalibani  and @RikoOrlando

**Pull request merged**